src/settings-ui/Settings.UI/SettingsXAML/Views/PowerLauncherPage.xaml

```diff
@@ -13,6 +14,293 @@
+            CheckBoxTemplate="{StaticResource CheckBoxTemplate}"
+            CheckBoxTextBoxTemplate="{StaticResource CheckBoxTextBoxTemplate}"
+            ComboBoxTemplate="{StaticResource ComboBoxTemplate}"
+            EmptyTemplate="{StaticResource EmptyTemplate}"
+            MultilineTextBoxTemplate="{StaticResource MultilineTextBoxTemplate}"
+            NumberBoxTemplate="{StaticResource NumberBoxTemplate}"
+            TextboxTemplate="{StaticResource TextBoxTemplate}" />
+
+        <DataTemplate x:Key="CheckBoxTemplate" x:DataType="ViewModels:PluginAdditionalOptionViewModel">
+            <StackPanel HorizontalAlignment="Stretch" Orientation="Vertical">
+                <tkcontrols:SettingsCard
+                    MinHeight="0"
+                    Margin="1"
+                    Padding="0,6,0,12"
+                    Background="Transparent"
+                    BorderThickness="0,0,0,0"
+                    ContentAlignment="Left"
+                    CornerRadius="0">
+                    <controls:CheckBoxWithDescriptionControl
+                        Margin="56,0,0,0"
+                        Description="{x:Bind Path=DisplayDescription}"
+                        Header="{x:Bind Path=DisplayLabel}"
+                        IsChecked="{x:Bind Value, Mode=TwoWay}" />
+                </tkcontrols:SettingsCard>
+                <Rectangle Style="{StaticResource OptionSeparator}" />
+            </StackPanel>
+        </DataTemplate>
+
+        <DataTemplate x:Key="ComboBoxTemplate" x:DataType="ViewModels:PluginAdditionalOptionViewModel">
+            <StackPanel HorizontalAlignment="Stretch" Orientation="Vertical">
+                <tkcontrols:SettingsCard
+                    MinHeight="0"
+                    Margin="56,8,45,8"
+                    Background="Transparent"
+                    BorderThickness="0,0,0,0"
+                    CornerRadius="0"
+                    Description="{x:Bind Path=DisplayDescription}"
+                    Header="{x:Bind Path=DisplayLabel}">
+                    <ComboBox
+                        MinWidth="{StaticResource SettingActionControlMinWidth}"
+                        DisplayMemberPath="Key"
+                        ItemsSource="{x:Bind Path=ComboBoxItems}"
+                        SelectedValue="{x:Bind ComboBoxValue, Mode=TwoWay}"
+                        SelectedValuePath="Value" />
+                </tkcontrols:SettingsCard>
+                <Rectangle Style="{StaticResource OptionSeparator}" />
+            </StackPanel>
+        </DataTemplate>
+
+        <DataTemplate x:Key="TextBoxTemplate" x:DataType="ViewModels:PluginAdditionalOptionViewModel">
+            <StackPanel HorizontalAlignment="Stretch" Orientation="Vertical">
+                <tkcontrols:SettingsCard
+                    MinHeight="0"
+                    Margin="56,8,45,8"
+                    Background="Transparent"
+                    BorderThickness="0,0,0,0"
```

Comment:
Why not use this for a bottom border?

Reply:
Gave a try but this works well only for templates that contains only the `SettingsCard`.
Some type of options are hard to test since they aren't used by built-in plugins so I prefer to keep visual unchanged.